gmapping/src/slam_gmapping.cpp

```diff
@@ -406,9 +406,9 @@ SlamGMapping::initMapper(const sensor_msgs::LaserScan& scan)
   }
 
   // Check that laserscan is from -x to x in angles:
-  if (fabs(fabs(scan.angle_min) - fabs(scan.angle_max)) > FLT_EPSILON)
+  if (fabs(fabs(scan.angle_min) - fabs(scan.angle_max)) > FLT_EPSILON)  
```

Comment:
Perhaps it is better to use this criteria:

if (fabs(fabs(scan.angle_min) - fabs(scan.angle_max)) > (scan.angle_increment * 2))